convention.md

```diff
@@ -91,7 +91,7 @@ Controllers are free to restrict discovery to a specific root topic, configurabl
 
 As soon as a device starts to publish any Homie related topic,
 it MUST finish with all topics within a timeframe of 500ms.
-Controller can assume that not received topics within this timeframe are not set.
+Controllers should assume an empty string for topics values not received within this timeframe.
```

Comment:
```suggestion
Controllers should assume the default for topic values not received within this timeframe.
```

Reply:
Up for discussion. Should we distinguish between mandatory and optional attributes?